github/issue.go

```diff
@@ -111,15 +111,31 @@ func (g GitHub) maybeOpinion(issueHook *octokat.IssueHook) error {
 	return nil
 }
 
+func labelFromVersion(version, suffix string) string {
+	switch suffix {
+	// Dev suffix is associated with a master build.
+	case "dev":
+		return "version/master"
```

Comment:
yes, that'd work, question is; is this label useful then, if it's not really used for filtering :smile: